src/database/migrations/postgres/20240314140913_accounts_table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TYPE account_origin AS ENUM ('cli', 'github');
+
+CREATE TABLE accounts(
+    id uuid NOT NULL,
```

Comment:
let's make this a DID `varchar`

Reply:
Sure, but this isn't a real accounts repository yet. I just played with something simple.